### PR TITLE
CF-446 access feed

### PR DESCRIPTION
**Pull request by ChandraAddala**
[CF-446] Create new feeds for Cloud Feeds user access events
     - New feed, feeds_access/events feed for feeds UAE
     - Added version attribute for auditData
     - Updated sample xmls with version attribute
     - Updated api-operations with the new feed for docs
     - Changes to make region/datacenter required.

### CODE REVIEW

**Comment by ChandraAddala** [comment]
@shintasmith @usnavi Pls review.

**Comment by ChandraAddala** [comment]
I am still working on the xinclude part., couldnt get it working.

**Comment by usnavi** [comment]
@ChandraAddala - did you do a PR on your own repo?!

**Comment by shintasmith** [comment]
looks good to me.... @usnavi, are you cool with the minOccurs="1" on region and DC?

File: core_xsd/uae.xsd [hunk 1]
```diff
@@ -19,27 +19,26 @@
     <xs:complexType name="AuditDataType">
         <xs:sequence>
             
-            <xs:element name="region" type="event:Region" default="GLOBAL" minOccurs="0" maxOccurs="1">
+            <xs:element name="region" type="event:Region" default="GLOBAL" minOccurs="1" maxOccurs="1">
```
Comment (usnavi) [inline core_xsd/uae.xsd]: What prompted this?  I think its the right move (screw this default stuff).  Should we remove default?

Comment (shintasmith) [inline core_xsd/uae.xsd]: @ChandraAddala told me yesterday that he tested sending UAE without the <ua:region> element. The value "GLOBAL" is not inserted if the element does not exist at all. It is only inserted if the element exists but has no other value. So if we want to have region=GLOBAL if publisher doesn't send the element, we have to write additional XSLT. 

I was telling Chandra that it's debatable whether we want to make UAE the same as Standard Events wrt GLOBAL region and DC.

Comment (shintasmith) [inline core_xsd/uae.xsd]: I don't want to write additional XSLT if we don't need it. So if you are ok with minOccurs=1, then lets go with that. 

If you want to even get rid of default=GLOBAL, I'm fine with it too. :-) Free world! No validation!

Comment (usnavi) [inline core_xsd/uae.xsd]: Since these events will all be generated by repose, I don't think supporting the default value is of much importance.

Comment (shintasmith) [inline core_xsd/uae.xsd]: These events are generated by Repose, true. But the value of those need to come from somewhere. Is the region/DC value coming from the template or from Repose itself?

Comment (usnavi) [inline core_xsd/uae.xsd]: The dc & region comes from repose and is referenced within the template through a  variable.,

**Pull request closed without merging**